### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

**Review by iceicery — CHANGES_REQUESTED** [review]
Thanks for putting everything together. Nice job! I am so happy to see our page. It looks great! While checking, I noticed the footer-list title is a little different from the design. Figma didn't provide the number. But I think color: rgba(255, 255, 255, 0.5); might work. What do you think?
image

File: src/components/SectionTitle/SectionTitle.css [hunk 1]
```diff
@@ -19,6 +19,7 @@
 .sectiontitle__subtitle {
   padding: 0;
   margin: 8px 0 0 0;
+  font-weight: 400;
```
[the remark this answers is not in the document]
Comment (julieg18) [reply src/components/SectionTitle/SectionTitle.css:22]: You are correct! It's better to place styles at the parent, but h3s have a default weight of bold, which overrode the parent. So I was forced to place a font-weight: 400 on it.

File: src/pages/index.js [hunk 2]
```diff
@@ -16,11 +22,21 @@ const IndexPage = () => {
...
       </Header>
       <main>
+        <section className="section enterprise">
+          <SectionTitle
+            title="Enterprise"
+            subtitle="AI hand in hand with software development. Built with data scientists, ML engineers, and data engineers in mind."
+          />
+          <Diagram />
+          <CompanyHighlightList />
+        </section>
```
[the remark this answers is not in the document]
Comment (julieg18) [reply src/pages/index.js:35]: Hmmm, are you sure? The section wouldn't have a title or a divider, making it different from all the other sections 🤔

File: src/components/SectionTitle/SectionTitle.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (iceicery) [reply src/components/SectionTitle/SectionTitle.css:22]: I see. Good catch!

File: src/pages/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (iceicery) [reply src/pages/index.js:35]: I mean like <Community/>: We could have <Enterprise/> <Career/> <Team/> <Leaders /> components just to make page simple to read.  I think both would work as long as we are consistent to all pages. What do you think?

[the remark this answers is not in the document]
Comment (julieg18) [reply src/pages/index.js:35]: Oh I see what you're saying! I'll make an Enterprise section!

**Review by iceicery — APPROVED** [review]
Great work!

**Pull request merged**